internal/commands/sbommonitor/sbommonitor.go

```diff
@@ -26,6 +30,28 @@ func RegisterWorkflows(e workflow.Engine) error {
 	return nil
 }
 
+func loadPolicyFile(sbomFilePath string) (string, error) {
```

Comment:
what about returning an `io.Reader` instead? or `[]byte`? seems like this would be a better fit than `string` for a function that's loading a file.